mobile-app/src/components/SettingsComponents.tsx

```diff
@@ -66,17 +67,17 @@ export function SettingsChoice({
   defaultSetting: number;
 }) {
   const [selected, setSelected] = useState(defaultSetting);
+  const [setting, setSetting] = useAtom(settingAtom(settingKey));
 
   useEffect(() => {
-    const savedSetting = SecureStore.getItem(settingKey);
-    if (savedSetting) {
-      setSelected(parseInt(savedSetting));
+    if (setting) {
+      setSelected(parseInt(setting));
     }
   });
 
   function changeSetting(index: number) {
     setSelected(index);
-    SecureStore.setItem(settingKey, index.toString());
+    setSetting(index.toString());
```

Comment:
we can just store the number directly now (`SecureStore.setItem` could only store strings). Will prevent all these goofy ahh type conversions.

Reply:
u gotta change the atom